test/test_hyper.py

```diff
@@ -1129,6 +1130,29 @@ def test_adapter_accept_client_certificate(self):
             'http',
             cert=CLIENT_PEM_FILE)
         assert conn1 is conn2
+        assert conn1._conn.ssl_context.check_hostname
+        assert conn1._conn.ssl_context.verify_mode == ssl.CERT_REQUIRED
+
+    def test_adapter_respects_disabled_ca_verification(self):
+        a = HTTP20Adapter()
+        conn = a.get_connection(
+            'http2bin.org',
+            80,
+            'http',
+            verify=False,
+            cert=CLIENT_PEM_FILE)
```

Comment:
I don't think so, I can't find it in the `SSLContext` API.
Maybe we could try an integration test instead?